### DISCUSSION TITLE
create a compiled CommonJS lib for users of node < 16

### DISCUSSION THREAD

**Discussion by jimhallup** [node-fetch/node-fetch#1499] [Ideas]
This should be relatively easy to achieve using `main` and `module` parameters in package.json

**Comment by jimmywarting** [comment]
we don't necessery want to have to reintroduce build steps and downlevel to commonjs any more
but we are thinking about creating a CommonJS wrapper 
https://github.com/node-fetch/node-fetch/issues/1327
